misc/kickstart_create_network_line.py

```diff
@@ -109,7 +105,8 @@ def find_usable_net_devs(location):
                 add = False
         if speed  <= 0:
             add = False
-        if TO_LOG: print add, dev
+        if TO_LOG:
+            print (add, dev)
```

Comment:
In python 2 this changes `(add, dev)` into a tuple which is then printed as a tuple. Why not just add `from __future__ import print_function` as the first line of code in the script and make it actually use the print function from python3? Ditto this comment to the other changes like this below.